.circleci/config.yml

```diff
@@ -67,14 +94,61 @@ jobs:
             -n "Release ${CIRCLE_TAG}" \
             -b "Generated by Instaclustr's Circle CI account." \
             ${CIRCLE_TAG} ./bin/
-
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.14
```

Comment:
(question) any reason why we are not building on go 1.16 ?

Reply:
This is what we have been using, and I want to avoid opportunistically upgrading the version and add unnecessary overhead given that this PR was already broken by number of other things